indicators/ETC-funccode.yml

```diff
@@ -0,0 +1,20 @@
+title: ETC_PhishingDetection
```

Comment:
The title ideally should follow the *unofficial* rule title standard i.e. ETC Phishing Kit followed by a unique identifier so that the kit can be correctly identified if there are multiple kits targeting ETC